### ISSUE TITLE
SigningError - Cannot read property getCredentials of undefined in google-auto-auth

### ISSUE THREAD

**Issue by drew-r** [googleapis/nodejs-storage#147]
Looks like release v0.9.6  over at https://github.com/stephenplusplus/google-auto-auth/ has broken something here
```
Cannot read property 'getCredentials' of undefined
{"name":"SigningError","stack":"SigningError\n    at ./node_modules/@google-cloud/storage/src/file.js:1715:16\n    at getCredentials (./node_modules/google-auto-auth/index.js:270:9)\n    at getAuthClient (./node_modules/google-auto-auth/index.js:148:9)\n    at <anonymous>"}}
```

**Comment by stephenplusplus** [comment]
Thanks for reporting. How are you authenticating?

**Comment by drew-r** [comment]
Explicit credentials using `projectId` and `keyFilename`

**Comment by stephenplusplus** [comment]
Alright, I've released a new version of google-auto-auth (0.9.7). Would you mind un- and re-installing your dependencies to see if we've found a fix?
